index.html

```diff
@@ -10,4 +10,10 @@
   <h2>Hello Kedivim girls and boys!</h2>
 </body>
 
+
+<footer>
+  <p>Author: Theodore Chaikalis</p>
+  <p><a href="mailto:info@teohaik.com">info@teohaik.com</a></p>
+</footer>
+
 </html>
```

Comment:
all files must have an empty line at the end. please add an empty line at the end of the file

Reply:
fixed